pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -271,17 +271,80 @@ def _on_evaluation_epoch_end(self) -> None:
         self.trainer._call_lightning_module_hook("on_epoch_end")
         self.trainer.logger_connector.on_epoch_end()
 
+    def _get_keys(self, iterable: Union[List, dict]) -> Iterable[str]:
+        if isinstance(iterable, list):
+            for i in iterable:
+                yield from self._get_keys(i)
+        elif isinstance(iterable, dict):
+            for k, v in iterable.items():
+                if isinstance(v, dict):
+                    yield from self._get_keys(v)
+                else:
+                    yield k
+
+    def _find_value(self, data: dict, target: str) -> Iterable[Any]:
+        for k, v in data.items():
+            if k == target:
+                yield v
+            elif isinstance(v, dict):
+                yield from self._find_value(v, target)
+
     def _print_results(self, results: List[_OUT_DICT], stage: RunningStage) -> None:
```

Comment:
Let's make it a staticmethod so we don't need to instantiate the class to run it during testing

Reply:
Ok, done!

Reply:
You missed `_print_results` itself :P

Reply:
Oh I see, fixed it!